packages/next/src/build/webpack/plugins/flight-manifest-plugin.ts

```diff
@@ -95,7 +109,7 @@ export interface ClientReferenceManifest extends ClientReferenceManifestForRsc {
     [moduleId: string]: ManifestNode
   }
   entryCSSFiles: {
-    [entry: string]: string[]
+    [entry: string]: CssResource[]
```

Comment:
I think it would be nice to have this stored in a separate manifest instead of mixing with current client reference manifest, since they're for different purpose. Client reference manifest can only keep the references and links, and css content might be larger and harder to read, also could have potentials to get optimized by any processor.

Reply:
I agree, but since the manifest is per-page, it’s less of a priority at the moment. There is already a plan to revisit inlining for the general case, so we can work on creating a better construct when we address that.